frontend/packages/dev-console/src/components/pipelineruns/PipelineRunResourceList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { DevPreviewBadge } from '@console/shared';
+import {
+  pipelineRunFilterReducer,
+  pipelineRunStatusFilter,
+} from '../../utils/pipeline-filter-reducer';
+import { PipelineRunModel } from '../../models';
+import PipelineRunsList from './PipelineRunList';
+
+const filters = [
+  {
+    type: 'pipelinerun-status',
+    selected: ['Succeeded'],
+    reducer: pipelineRunFilterReducer,
+    items: [
+      { id: 'Succeeded', title: 'Complete' },
+      { id: 'Failed', title: 'Failed' },
+      { id: 'Running', title: 'Running' },
```

Comment:
I think we need to start making concerted efforts to avoid any `id` or reference value being a magic string. It's not great for maintenance moving forward. I'd like to see an `enum` be used for the filter ids and in a common file that can be used for other filters that match this design.

For instance - there is similar logic found here: `frontend/packages/dev-console/src/components/pipelines/PipelineAugmentRuns.tsx`

Reply:
Using enums now